tests/test_hello.py

```diff
@@ -0,0 +1,19 @@
+"""File tests hello.py file."""
```

Comment:
hello.py and its test are unrelated to the Slack/Trello chatbot feature. Consider removing them